packages/smooth_app/lib/pages/user_preferences_page.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:provider/provider.dart';
+import 'package:smooth_app/data_models/product_preferences.dart';
+import 'package:smooth_app/data_models/user_preferences.dart';
+import 'package:smooth_app/generic_lib/design_constants.dart';
+import 'package:smooth_app/pages/preferences/abstract_user_preferences.dart';
+import 'package:smooth_app/pages/preferences/user_preferences_dev_mode.dart';
+import 'package:smooth_app/pages/preferences/user_preferences_food.dart';
+import 'package:smooth_app/pages/preferences/user_preferences_profile.dart';
+import 'package:smooth_app/pages/preferences/user_preferences_settings.dart';
+import 'package:smooth_app/themes/theme_provider.dart';
+
+/// Preferences page for attribute importance
+class UserPreferencesPage extends StatefulWidget {
+  const UserPreferencesPage();
+
+  @override
+  State<UserPreferencesPage> createState() => _UserPreferencesPageState();
+}
+
+class _UserPreferencesPageState extends State<UserPreferencesPage> {
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+    final UserPreferences userPreferences = context.watch<UserPreferences>();
+    final ThemeProvider themeProvider = context.watch<ThemeProvider>();
+    final ThemeData themeData = Theme.of(context);
+    final ProductPreferences productPreferences =
+        context.watch<ProductPreferences>();
+
+    final List<AbstractUserPreferences> items = <AbstractUserPreferences>[
+      UserPreferencesProfile(
+        setState: setState,
+        context: context,
+        userPreferences: userPreferences,
+        appLocalizations: appLocalizations,
+        themeData: themeData,
+      ),
+      UserPreferencesFood(
+        productPreferences: productPreferences,
+        setState: setState,
+        context: context,
+        userPreferences: userPreferences,
+        appLocalizations: appLocalizations,
+        themeData: themeData,
+      ),
+      UserPreferencesSettings(
+        themeProvider: themeProvider,
+        setState: setState,
+        context: context,
+        userPreferences: userPreferences,
+        appLocalizations: appLocalizations,
+        themeData: themeData,
+      ),
+    ];
+    items.add(
```

Comment:
I don't think so: only developers should see that.

Reply:
@monsieurtanuki Do you mean the status of the migration shouldn't be in the dev settings?

Reply:
@g123k I mean that dev settings is for developers only, with specific features that mean nothing to the end-users.
The migration is more like an _advanced_ feature for end-users.

There should probably be
* a specific item for every user saying "Upgraded version of OFF with 753 products migrated"
* a specific item for developers like "Restart the migration for debug purpose"

Reply:
What would be a suitable place for that information? In App Settings?

Reply:
Yes, let's try in App Settings. The goal now is to merge quickly while keeping the dev mode private to developers, in order to make tests possible.
Later we'll probably say "let's put it there instead", or "migration deserves its own block", but for the moment App Settings looks appropriate enough.